src/Cake.GitHub/Issues/Exceptions/GitHubIssueException.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Cake.GitHub
+{
+    /// <summary>
+    /// Base class for exceptions that can be thrown while updating GitHub Issues (or Pull Requests)
+    /// </summary>
+    public abstract class GitHubIssueException : Exception
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GitHubIssueException"/> class with the specified error message.
+        /// </summary>
+        protected GitHubIssueException(string message) : base(message)
```

Comment:
I prefer base calls on a new line, but it's a personal preference. Don't know the exact naming convention used in cake development.

Reply:
Fixed with 19091058f0a2eeabbfb231421b3bdeb67231bd1c